tsconfig.json

```diff
@@ -23,9 +23,5 @@
   "vueCompilerOptions": {
     "plugins": ["@vue/language-plugin-pug"]
   },
-  "include": [
-    "app/javascript/**/*.ts",
-    "app/javascript/**/*.d.ts",
-    "app/javascript/**/*.vue"
-  ]
+  "include": ["app/javascript/**/*.ts", "app/javascript/**/*.vue"]
```

Comment:
Mostly unrelated change, but I happened to notice that `app/javascript/**/*.ts` includes `app/javascript/**/*.d.ts`, so we shouldn't need to list them both.